File: PHP Word Examples/WatermarksApiDemo/setImageWatermark.php
<?php

use Spire\Cloud\Word\Sdk\Client\Api\WatermarksApi;
use Spire\Cloud\Word\Sdk\Client\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new WatermarksApi($configuration);

$name = "Template.docx";
$image_path = "images/image.png";
$folder = "input";
$storage = null;
$scaling = 120;
$washout = true;
$password = null;
$dest_file_path =  "Watermark/SetImageWatermark_true.docx";
$apiInstance->setImageWatermark($name, $image_path, $dest_file_path, $folder, $storage, $scaling, $washout, $password);

File: Python Word Examples/WatermarksApiDemo/setImageWatermark.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.WatermarksApi(configuration)

filename = 'Template.docx'
inputimg = 'images/Background.png'
#Just moving "'password': '123'" if there is no password.
extra = {'folder': "input",'scaling': 120,'washout': True,'password': '123','dest_file_path': 'output/SetImageWatermark.docx'}
result = api.set_image_watermark(filename, inputimg, **extra)